my-app/src/components/userCard/user-card.component.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./user-card.css";
+import Name from "../name/name.component";
+import Avatar from "../avatar/avatar.component";
+import Role from "../role/role.component";
+import UserCardSection from "../userCardSection/user-card-section.component";
+
+const UserCard = ({ name, mentor, mentee, sections }) => {
```

Comment:
I see sections and name are not marked as required. Have you tried omitting them or setting null/undefined? It is possible there might be some errors or undefined printed to the user